app/views/layouts/application.html.erb

```diff
@@ -12,8 +12,16 @@
   <meta name="theme-color" content="#d6d7d9">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <%= tag.link href: asset_path('ustranscom-emblem-256x256.png'), rel: 'apple-shortcut-icon' %>
-  <%= tag.link href: asset_path('ustranscom-emblem-256x256.png'), rel: 'shortcut icon' %>
+  <!-- PNG versions of favicon.ico -->
+  <link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
+  <link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
+  <!-- iOS homescreen icon -->
+  <link rel="apple-touch-icon" href="/apple-touch-icon.png">
```

Comment:
[Some advocate generating every possible resolution under the sun for iOS](https://css-tricks.com/favicon-quiz/). That sounded like a lot of work. 180x180 will have to suffice (rescaled automatically) for everyone.